src/features/offer/components/__tests__/ReportOffer.native.test.tsx

```diff
@@ -159,4 +159,20 @@ describe('ReportOffer', () => {
       fireEvent.press(reportOfferButton)
       expect(hideReportModal).toHaveBeenCalledWith()
     })
+    describe('Report offer description', () => {
+      const ReportOfferFirstModalComponent = render(
```

Comment:
✅  tu peux mettre un it(), ça donnera un nom à ton snapshot

Reply:
top